jams.js

```diff
@@ -36,6 +36,12 @@ const _jams =ast=> {
             return _jams(ast.children[0])
         }
         case 'str': {
+            if (ast.children.length == 1) {
```

Comment:
Should assert it’s a quoted string here and extract text, I guess, since it is not a full jams sub object

Reply:
Gonna merge as is but pls update

Reply:
Actually nvm it is fine to depend on the grammar as we do in other places 

Later we will make a version of this with a manual parser with better errors, there it will matter